projects/fsastorefrontlib/src/core/product-assignment/facade/product-assignment.service.ts

```diff
@@ -71,10 +71,9 @@ export class ProductAssignmentService {
 
   isUserAdminOfUnit(unitId: string): Observable<boolean> {
     return this.userService.get().pipe(
+      filter(user => user?.uid !== OCC_USER_ID_ANONYMOUS),
       map(user => {
-        if (user && user.uid !== OCC_USER_ID_ANONYMOUS) {
-          return (<B2BUser>user).orgUnit.uid === unitId;
-        }
+        return (<B2BUser>user).orgUnit?.uid === unitId;
```

Comment:
Added conditional operator to prevent browser console error after B2B user logs out.